src/zope/testrunner/formatter.py

```diff
@@ -52,7 +52,7 @@ class OutputFormatter:
 
     def __init__(self, options):
         self.options = options
-        self.last_width = 0
+        self.test_width = self.last_width = 0
         self.compute_max_width()
```

Comment:
TBH I don't know if it's enough to initialize test_width here, or if it should also be reset to 0 in every `test_success()`/`test_error()`/`test_failure()`.